### ISSUE TITLE
tonos-cli truncates leading zeros when issuing the result of getCustodians

### ISSUE THREAD

**Issue by asnov** [tonlabs/tonos-cli#46]
I'm using
```sh
tonos-cli --url main.ton.dev run 0:ebaa9810c3bd1fb7f02901509735ac287362b741e9e874ea2a36f94351e2c255 getCustodians '{}' --abi ../configs/SafeMultisigWallet.abi.json
```
It gives me:
```
"custodians": [
    {
      "index": "0x1",
      "pubkey": "0xf07a7cb924c7420520d0d98afad87d9b5e1765920fda698c22da6d0cd3354b9"
    },
    {
      "index": "0x6",
      "pubkey": "0x1a99622e54b4e87d603dd87c9cc936b388b2a0e1979bb56d4039cfad0fbadc8c"
    },
```
As you can see pubkey with index 0x01 has 63 characters instead of 64. The leading zero was cutted out.
At the same time Public Key was entered correctly in the command to create this SafeMultisigWallet. Please, check the screenshot bellow:

![image](https://user-images.githubusercontent.com/9980139/95515184-95066980-0993-11eb-85e2-0bd83b31a4f1.png)

**Comment by joydark** [comment]
Sorry for late response.
0x - return in hex where it's expected behavior to leading zero to be trimmed.

**Comment by asnov** [comment]
@joydark,
Expected behavior is to trim leading zero for decimal numbers because decimal numbers starting with zero means octal numbers in some languages.
Do you know why hexadecimal numbers start with "0x"? It is because they store not only the value, but also the digit capacity. 0x1 is not equal 0x0001 because the last one could case overflow in some architectures. So trimming leading zeros for hexadecimals could NOT be expected behaviour. It is a BUG.
Could you be so kind to register it as a bug so the guys could fix it?

**Comment by joydark** [comment]
The issue is part of SDK and it's already solved in new version. We are in the process of migrating to it in https://github.com/tonlabs/tonos-cli/tree/sdk-1-0

**Comment by asnov** [comment]
So if this is solved already in sdk 1.0 then it is not _this-is-not-a-bug-but-a-feature_ thing as you said before.
It looks like I should I wait for sdk-1-0, right? When do you expect it could be released?

**Comment by asnov** [comment]
fixed in tonos_cli 0.2.0
COMMIT_ID: acdabec127e19b2c91781439178d6d683f3a926a
BUILD_DATE: 2021-01-04 18:03:43 -0300
COMMIT_DATE: 2021-01-04 17:30:41 -0300
GIT_BRANCH: master

the same command gives me:
```
"custodians": [
    {
      "index": "1",
      "pubkey": "0x0f07a7cb924c7420520d0d98afad87d9b5e1765920fda698c22da6d0cd3354b9"
    },
```
